protobufs/livekit_metrics.proto

```diff
@@ -0,0 +1,55 @@
+syntax = "proto3";
+
+package livekit;
+option go_package = "github.com/livekit/protocol/livekit";
+option csharp_namespace = "LiveKit.Proto";
+option ruby_package = "LiveKit::Proto";
+
+
+message MetricsBatch {
+  repeated TimeSeriesMetric time_series = 1;
+  repeated AnnotationMetric annotations = 2;
+}
+
+message TimeSeriesMetric {
+  // Metric name, e.g., "speech_probability"
+  string label = 1;
+  // optional participant identity
+  string participant_identity = 2;
+  // optional track sid
+  string track_sid = 3;
+  // if track_sid is set, timestamp represents the RTP timestamp
+  int64 start_timestamp = 4;
```

Comment:
is the implication that, as-is, this is wall clock if track_sid isn't set? if so - i agree with Neil it would be good to always have the wall clock value

Reply:
agreed -- have a timestamp, and optionally a (`track_sid` + `track_timestamp`)

Reply:
Agree that it may be confusing for a same field to have 2 different clocks.
Looking at the current code, if we have to add the wall clock, we would need to duplicate a bunch of fields (start_timestamp, end_timestamp, timestamp of values)

Do you think the wall clock is necessary when we already know the start_time of a room, or the published time of a track for e.g? Also wondering about the use cases on when you really need the wall clock of your annotated data?